draft-ietf-oauth-device-flow.xml

```diff
@@ -238,14 +238,11 @@
 
 	<t>The client initiates the flow by requesting a set of verification
 	codes from the authorization server by making an HTTP "POST" request
-	to the device endpoint.  The client constructs a request URI by
+	to the device authorization endpoint.  The client constructs a request URI by
```

Comment:
I don't think this is a request URI - It's just a request posted at the device authorization endpoint.  Delete "URI".